src/components/molecules/ScheduleItemNG/ScheduleItemNG.tsx

```diff
@@ -1,46 +1,147 @@
+
+      const eventLink = getFullVenueInsideUrl(roomUrlParam);
+      navigator.clipboard.writeText(eventLink);
+    },
+    [roomUrlParam]
+  );
+
+  const goToEventLocation = useCallback(() => {
+    if (!eventRoom) {
+      openUrl(roomUrlParam);
+
+      return;
+    }
+
+    if (event.room) {
+      enterRoom();
+    } else {
+      enterVenue(event.venueId);
+    }
+  }, [enterRoom, event, eventRoom, roomUrlParam]);
+
+  const infoContaier = classNames("ScheduleItemNG__info", {
+    "ScheduleItemNG__info--active": isCurrentEventLive,
+  });
+
+  const timeContainer = classNames("ScheduleItemNG__time", {
+    "ScheduleItemNG__time--live": isCurrentEventLive,
+  });
 
   return (
-    <div className="ScheduleItemNG">
-      <div className="ScheduleItemNG--info">
-        <span className="ScheduleItemNG--date">
-          {showDate &&
-            formatDateRelativeToNow(eventStartTime(event), formatDateOptions)}
+    <div className="ScheduleItemNG" onClick={toggleEventExpand}>
+      <div className={infoContaier}>
+        <span className="ScheduleItemNG__date">
+          {!isCurrentEventLive &&
+            showDate &&
+            formatDateRelativeToNow(eventStartTime(event))}
         </span>
 
-        <span className="ScheduleItemNG--time">
-          {formatTimeLocalised(eventStartTime(event))}
+        <span className={timeContainer}>
+          {isCurrentEventLive
+            ? "Live"
+            : formatTimeLocalised(eventStartTime(event))}
         </span>
 
-        <span className="ScheduleItemNG--date">
-          {showDate &&
-            formatDateRelativeToNow(eventEndTime(event), formatDateOptions)}
+        <span className="ScheduleItemNG__date ScheduleItemNG__time--end">
+          {showDate && formatDateRelativeToNow(eventEndTime(event))}
         </span>
 
-        <span className="ScheduleItemNG--time">
+        <span className="ScheduleItemNG__time ScheduleItemNG__time--end">
           {formatTimeLocalised(eventEndTime(event))}
         </span>
       </div>
-      <div className="ScheduleItemNG--name">{event.name}</div>
+      <div className="ScheduleItemNG__details">
+        <div className="ScheduleItemNG__name">{event.name}</div>
+        <div className="ScheduleItemNG__place">
+          <span className="ScheduleItemNG__place--location">in</span>{" "}
+          {roomUrlParam}
+        </div>
+        {isEventExpanded && (
+          <>
+            <div className="ScheduleItemNG__description">
+              {event.description}
+            </div>
+            <div className="ScheduleItemNG__buttons">
+              <Button
+                customClass="ScheduleItemNG__button ScheduleItemNG__button--copy"
+                onClick={handleCopyEventLink}
+              >
+                Copy event link
+              </Button>
+              <Button
+                customClass="ScheduleItemNG__button"
+                onClick={goToEventLocation}
+              >
+                See on playa
```

Comment:
This should say "see on map" for staging and "see on playa" is only for the bm environment

Reply:
fixed in e8b364b95eb450911b765952bddc625b416964b5